load_dumps.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require('app/config.php');
+
+$db = new PDO('mysql:dbname='.DB_NAME, DB_USER, DB_PASS);
+
+$sql = file_get_contents('databases/saathi.sql');
+$qr = $db->exec($sql);
```

Comment:
Why are you assigning a variable if its not being used. I think you can check for the success and display any error.
<pre>
if ($qr) {
    // Report Success
} else {
    // Report Failure
}
</pre>